tests/test_custom_rnncell.py

```diff
@@ -284,6 +284,7 @@ def test_multi_rnn_lstm(self, state_is_tuple=True):
 
     @check_opset_min_version(8, "Scan")
     @check_tf_min_version("1.8")
+    @skip_specific_opset_version(9, "ReverseSequence cannot be efficient mapped in OPSET 9")
```

Comment:
just "ReverseSequence not supported" like others above? there is enough information in reverse_op9